packages/docs/vite.config.ts

```diff
@@ -87,31 +87,48 @@ export default defineConfig(({ command, mode }) => {
       // https://github.com/hannoeru/vite-plugin-pages
       Pages({
         extensions: ['vue', 'md'],
-        pagesDir: [
-          { dir: 'src/pages', baseRoute: '' },
+        dirs: [
+          { dir: 'src/pages', baseRoute: 'pages' },
           { dir: 'src/api', baseRoute: 'api' },
+          { dir: 'src/wireframes', baseRoute: 'wireframes' },
         ],
         extendRoute (route) {
-          if (['index', 'all'].includes(route.name)) {
+          if (['index', 'all'].includes(route.component)) {
```

Comment:
This never matches, and probably didn't before either.

Reply:
double check. It didn't work with route.name, but route.component uses the actual index.md filename.

Reply:
yeah but it's never just `'index'` or `'all'`